src/domain/models/customer.py

```diff
@@ -0,0 +1,13 @@
+from pydantic import BaseModel
+
+
+class CustomerRequest(BaseModel):
+    username: str
+    email: str
+
+
+class CustomerResponse(BaseModel):
+    id: int
```

Comment:
Nesse caso aqui quando for adicionado o linter vai apontar que o campo "id" está obscurecendo um atributo nativo do python, por isso seria bom adicionar a classe `Field` do Pydantic e colocar um alias, ficaria mais ou menos assim:

`
class CustomerRequest(BaseModel):
       id_: int = Field(alias='id')
       username: str
       email: str
`